.gitattributes

```diff
@@ -0,0 +1,7 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
```

Comment:
- [ ] good job the app actually is working 
![image](https://user-images.githubusercontent.com/74744787/211169501-71e6c6ab-414e-43bb-802e-8dd8b5fe41d4.png)

but I noticed that you didn't add the URL for the front end in the backend repo readme file kindly add it :+1: